### PR TITLE
Serve static

### PR DESCRIPTION
**Pull request by coffeegoddd**
Added server to serve static file, changed script from "build" for webpack to "react-dev".

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,8 @@
   "main": "webpack.config.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "build": "webpack -d --watch"
+    "react-dev": "webpack -d --watch",
+    "server-dev": "nodemon server/index.js"
```
Comment (smopart) [inline package.json:9]: Do we need server-dev script if this is the package.json for the UI?

File: package.json [hunk 2]
```diff
@@ -26,7 +27,9 @@
     "babel-cli": "^6.26.0",
     "babel-loader": "^8.0.5",
     "babel-preset-env": "^1.7.0",
+    "body-parser": "^1.18.3",
     "css-loader": "^2.1.0",
+    "express": "^4.16.4",
```
Comment (smopart) [inline package.json:32]: might not be needed in the client package.json

File: package.json [hunk 3]
```diff
@@ -26,7 +27,9 @@
     "babel-cli": "^6.26.0",
     "babel-loader": "^8.0.5",
     "babel-preset-env": "^1.7.0",
+    "body-parser": "^1.18.3",
```
Comment (smopart) [inline package.json:30]: might not be needed in client package.json

**Review by smopart — COMMENTED** [review]
possible updates to client package.json script

File: package.json (hunk 1 above)
Comment (coffeegoddd) [reply package.json:9]: We need a server to serve the bundle file. The bundle isn't in our app server, it's in this UI repo.

Comment (smopart) [reply package.json:9]: Thanks sounds good. will merge pull request

**Pull request merged**